jumpssh/session.py

```diff
@@ -581,6 +581,9 @@ def put(self,
         logger.debug("Copy local file '%s' on remote host '%s' in '%s' as '%s'"
                      % (local_path, self.host, remote_path, self.username))
 
+        # ensure the connection is open
+        #self.open()
```

Comment:
thanks for the fix, this is indeed missing but I would rather directly add the call to `self.open()` in the `SSHSession.file` method as this is the one accessing `paramiko.SFTPClient`

Reply:
done